deploy/src/main/resources/production-marathon-prerequisites/README.md

```diff
@@ -0,0 +1,36 @@
+# Production Marathon Prerequisites
+
+The Production Marathon Prerequisites deployment supports the deployment of the Job Service Database on Mesos/Marathon. This folder contains the marathon environment and template files that are required to deploy the Job Service Database.
+
+## Service Configuration
+
+### Marathon Template
+The `marathon.json.b` template file describes the marathon deployment information required for starting the Job Service Database. The template file uses property substitution to get values for configurable properties **required** for service deployment. These properties are configured in the marathon environment file `marathon.env`.
+
+### Marathon Environment
+The `marathon.env` file supports configurable property settings necessary for service deployment. These include:
+
+- `JOB_SERVICE_DB_SERVICE_PORT`: This configures the external port number on the host machine that will be forwarded to the Job Service Database containers internal 5432 port. This port is used to connect to the Job Service Database.
```

Comment:
Why is this not named 'POSTGRES_JOB_SERVICE_DB_PORT' to be consistent with user and password? It seems a little confusing to me.